lib/components/src/blocks/index.ts

```diff
@@ -1,3 +1,6 @@
+export * from './EmptyBlock';
+export * from './ZoomContext';
```

Comment:
Why export these?

Reply:
I needed them while trying to reproduce the issue with custom components.
It doesn't hurt to share them, but I could remove that commit from the PR if needed.